File: 1929-concatenation-of-array/1929-concatenation-of-array.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int length = nums.length;
        int[] ans = new int[length*2];
        int i=0,j=length;
        while(i<length && j<length*2){
            ans[i]=nums[i];
            ans[j]=nums[i];
            i++;
            j++;
        }
        return ans;
    }
}

File: 1929-concatenation-of-array/1929-concatenation-of-array.cpp
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        vector<int> ans(nums);
        ans.insert(ans.begin(),nums.begin(), nums.end());
//        copy(nums.begin(), nums.end(), back_inserter(ans));
        return ans; 
    }
};